portal/gil/templates/gil/base.html

```diff
@@ -44,6 +44,23 @@
         {{_("Loading")}}... &nbsp;
   </div>
   <section id="mainHolder">
+    <input type="hidden" id="currentUserId" value="{{user.id if user}}" />
+    {% if user and (user.has_role(ROLE.PATIENT) or user.has_role(ROLE.PARTNER)) %}
+      <input type="hidden" id="interventionMenu" value="true" />
+    {% endif %}
+    {% if timed_out %}
+      <input type="hidden" id="sessionTimedOut" value="true" />
+    {% endif %}
+    {% if request.args.get("disableLinks", "") %}
+      <input type="hidden" id="disableLinks" value="true" />
+    {% endif %}
+    {% if user and user.has_role(ROLE.CONTENT_MANAGER) %}
+      <input type="hidden" id="isContentManager" value="true" />
```

Comment:
This might be better generalized to query `api/user/{user_id}/roles` store a list of the current user's role on the frontend (localstorage?). 

Not a big deal, but worth considering if we need more role-specific logic on the frontend